### PR TITLE
Fix documentation page link in Indirect Data Analysis

### LINKED ISSUE #32403: Missing help page for Indirect Data Analysis
**Describe the bug**
Help button on Inelastic Data Analysis window opens the help window with the following message:
**Missing page - qthelp://org.mantidproject/doc/interfaces/indirect/Indirect Data Analysis.html**

**To Reproduce**
1. Go to Interfaces>Indirect>Data Analysis
2. Click the help button at the bottom

**Expected behavior**
Relevant help page should appear.

### PR DESCRIPTION
**Pull request by martyngigg**
**Description of work.**

Indirect data analysis was renamed some time ago and the documentation link got outdated. This fixes it.

**To test:**

* Build the `qthelp` target
* Open Interfaces->Indirect->Data Analysis
* Click the `?` button and the correct page should pop up.

Fixes #32403 . 

*This does not require release notes* because **it fixes a regression we made in this dev cycle**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Comment by RichardWaiteSTFC** [comment]
I've built qthelp but still get a missing page (looking for the old file)
`Missing page - qthelp://org.mantidproject/doc/interfaces/indirect/Indirect Data Analysis.html`
I note that there is a Indirect Data Analysis .rst.txt file here in the build folder 
`C:\mantid\build\docs\html\_sources\interfaces\indirect\Indirect Data Analysis.rst.txt`
so has the name recently changed?

**Review by RichardWaiteSTFC — APPROVED** [review]
Build all fixed the missing page - sorry!

File: qt/scientific_interfaces/Indirect/IndirectDataAnalysis.cpp [hunk 1]
```diff
@@ -116,7 +116,7 @@ void IndirectDataAnalysis::loadSettings() {
  */
 void IndirectDataAnalysis::tabChanged(int) {}
 
-std::string IndirectDataAnalysis::documentationPage() const { return "Indirect Data Analysis"; }
+std::string IndirectDataAnalysis::documentationPage() const { return "Inelastic Data Analysis"; }
```
Comment (gvardany) [inline qt/scientific_interfaces/Indirect/IndirectDataAnalysis.cpp:119]: Out of curiosity, while the title of the interface is now inelastic, the menu item is still in Indirect, is this expected? Was it not the plan to rename also the menu entry?

Comment (martyngigg) [reply qt/scientific_interfaces/Indirect/IndirectDataAnalysis.cpp:119]: Good spot. @Harrietbrown was the menu item going to change at some point?

Comment (Harrietbrown) [reply qt/scientific_interfaces/Indirect/IndirectDataAnalysis.cpp:119]: The plan is to eventualy have access to the interface in both the direct and indirect menus.

**Pull request merged**